docker-compose.yml

```diff
@@ -57,6 +57,31 @@ services:
       - "traefik.http.routers.backend-secured.rule=Host(`api.${HOST_NAME}`)"
       - "traefik.http.routers.backend-secured.tls.certresolver=mytlschallenge"
       - "traefik.docker.network=traefik"
+  rust_backend:
+    build:
+      context: ./rust_backend
+      target: ${APP_ENVIRONMENT}
+      args:
+        - APP_ENV=${APP_ENVIRONMENT}
+    container_name: rust_backend
+    restart: always
+    ports:
+      - 9000:9000
+    links:
+      - redis
```

Comment:
I don't think `links` is needed since they are both on the `traefik´ network.

Reply:
Done